content/about/index.md

```diff
@@ -10,4 +10,32 @@ Packit provides tooling and automation to integrate upstream open source project
 
 https://github.com/packit-service/packit
 
+
+### The key principles of packit
+
+* Reuse existing tools and services: don't reinvent the wheel.
```

Comment:
We should make this true..;-)

Reply:
ok, I'll update

Reply:
I didn't expect the change in text but on our side to do it....;-)

Reply:
and where do we reinvent?

Reply:
I've been thinking about overlap with tito/rdopkg/rpkg, but I understand that they're hard to incorporate and out of our control...

If I found anything that we can seriously use, I'll write it in my WIP research...

Reply:
Yes, please!

Reply:
(this is why I reworded to: "where it makes sense") :P